schedule/virt_autotest/kubevirt-tests-server.yaml

```diff
@@ -0,0 +1,29 @@
+name:           kubevirt-tests-server
```

Comment:
Could you help also move this file also into a 'kubevirt folder' in data/virt_autotest if this belongs to kubevirt or something else ? You should know that the data/virt_autotest folder starts growing, it would be better to curb potential chaos before it becomes true. We already have some too many different things just placed directly in this top folder.

Reply:
Referring to the comments:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14364#discussion_r828932026
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14364#discussion_r829628524